src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -803,17 +801,16 @@ private ValueSpec(Supplier<?> supplier, Predicate<Object> validator, BuilderCont
         public Object getDefault() { return supplier.get(); }
     }
 
-    public static class ConfigValue<T> implements Supplier<T>
-    {
-        private static boolean USE_CACHES = true;
+    public static class ConfigValue<T> implements Supplier<T> {
+        private final static boolean USE_CACHES = true;
```

Comment:
Minor: final after static
```suggestion
        private static final boolean USE_CACHES = true;
```

Reply:
why i didn't spotted it before?